packages/manager/core/api/src/filters.ts

```diff
@@ -67,10 +68,24 @@ export function applyFilters<T>(items: T[] = [], filters: Filter[] = []) {
           keep = keep && `${value}`.toLowerCase().endsWith(comp.toLowerCase());
           break;
         case FilterComparator.IsEqual:
-          keep = keep && `${value}`.toLowerCase() === comp.toLowerCase();
+          if (filter.type === FilterTypeCategories.Date) {
+            keep =
+              keep &&
+              new Date(`${value}`).setHours(0, 0, 0, 0) ===
```

Comment:
you can use startOfDay from date fns instead

Reply:
since this dependency is built and bundled, adding dependencies have an additional cost on the bundle size so i would advocate to do so only if it is mandatory

Reply:
Or maybe we could add tree shaking since date fns greatly supports it to avoid reinventing the wheel

Reply:
as you can see here, tree shaking have already been considered by specifying the sideEffect attribute in the package json of this package : https://github.com/ovh/manager/blob/master/packages/manager/core/api/package.json#L7

i don't think the wheel have been reinvented in this particular case, but it should be interesting to see the cost of importing the startOfDay function and see the impact on the whole bundle size

Reply:
Guys, it's not like I was coding a complex function with a complex algorithm... It's just the use of `setHours` which has the same effect as `startOfDay` without the need to add a dependency